pyvista/plotting/renderer.py

```diff
@@ -1307,9 +1307,17 @@ def reset_camera(self, render=True):
         ----------
         render : bool
             Trigger a render after resetting the camera.
+        bounds : iterable(int)
+            Automatically set up the camera based on a specified bounding box
+            ``(xmin, xmax, ymin, ymax, zmin, zmax)``.
 
         """
-        self.ResetCamera()
+        if isinstance(bounds, np.ndarray):
+            bounds = bounds.tolist()
+        if bounds:
```

Comment:
```suggestion
        if bounds is not None:
```
Better to check if it's not `None` and then proceed rather than converting to a list.  That way we save two lines of code and avoid having to make a test case for a `numpy` array.

Reply:
You're right!